index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="header">
```

Comment:
I think it would be better to use "header" semantic tag for better professionalism

Reply:
Hi, @Samuelwanza!! 👋
- That's a nice suggestion, you performed a better review.
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.